src/level_set/fortran/levelsetops2d.f.m4

```diff
@@ -556,6 +610,67 @@ c
 
       return
       end
+
+ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
+c
+c     Carry out relaxation scheme using a single time step of Runge-Kutta 
+c     scheme. Note that this a special case of the Gauss-Seidel sweeps.
+c
+c     Uses second order ENO for spatial discretization with a subcell
+c     fix near the interface
+c
+ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
+c
+      subroutine rkls3rdordereno2d(
+     &     U,U_gcw,
+     &     V,V_gcw,
+     &     ilower0,iupper0,
+     &     ilower1,iupper1,
+     &     dx,
+     &     stage,
+     &     use_subcell,
+     &     use_sign_fix)
+c
+      implicit none
+include(TOP_SRCDIR/src/fortran/const.i)dnl
+
+c
+c     Input.
+c
+      INTEGER ilower0,iupper0
+      INTEGER ilower1,iupper1
+      INTEGER U_gcw,V_gcw
+      INTEGER stage
+      INTEGER use_subcell,use_sign_fix 
+
+c
+c     Input/Output.
+c
+      REAL U(CELL2d(ilower,iupper,U_gcw))
+      REAL V(CELL2d(ilower,iupper,V_gcw))
+      REAL dx(0:NDIM-1)
+c
+c     Local variables.
+c
+      INTEGER i0,i1
+
+c     Prevent compiler warning for unused variable
+
+      stage = stage      
+
+      do i1 = ilower1,iupper1
+        do i0 = ilower0,iupper0
+            call evalrelax3rdordereno2d(U,U_gcw,V,V_gcw,
+     &                                  ilower0,iupper0,
+     &                                  ilower1,iupper1,
+     &                                  i0,i1,dx,
+     &                                  use_subcell,use_sign_fix)
+        enddo
+      enddo
+
+      return
+      end  
```

Comment:
same here - what's the difference between this one and `relaxationls3rdordereno2d` with `dir = 0`?

Reply:
Not for RK1 and RK2.